addon/hint/css-hint.js

```diff
@@ -41,7 +41,7 @@
     function add(keywords) {
       for (var name in keywords)
         if (!word || name.lastIndexOf(word, 0) == 0)
-          result.push(name);
```

Comment:
This definitely won't work for `pseudoClasses` defined above. What is it intended to do?

Reply:
CSS is not case-sensitive. The current solution doesn't work with properties and values that the user will camelCase (ex. [`optimizeSpeed`](https://developer.mozilla.org/en-US/docs/Web/CSS/text-rendering#values) instead of `optimizespeed`). I want to fix this. Do you have any idea how to solve the problem with `pseudoClasses`?

Reply:
No I mean, given that `keySet` is only called with lower-case strings, what is the purpose of making the object values hold another copy of the string?